src/Elastic/Indexer/Indexer.php

```
@@ -34,24 +34,12 @@ class Indexer {
 	 */
 	const REQUIRE_SAFE_REPLICATION = 'replication/safe';
 
-	/**
-	 * @var FileIndexer
-	 */
-	private $fileIndexer;
+	private ?FileIndexer $fileIndexer = null;
```

PHPMD (2.15.0)

[warning] Avoid unused private fields such as '$fileIndexer'. (undefined)

(UnusedPrivateField)